tests/test_edflow.py

```diff
@@ -0,0 +1,221 @@
+import pytest
+from edflow.iterators.tf_evaluator import TFBaseEvaluator
+from edflow.iterators.batches import DatasetMixin
+import tensorflow as tf
+import numpy as np
+import subprocess
+from tensorflow.contrib.framework.python.ops import add_arg_scope, arg_scope
+import subprocess
+import os
+import errno, sys
+
+
+class Model(object):
+    def __init__(self, config):
+        self.config = config
+
+
+class Iterator1(TFBaseEvaluator):
+    def __init__(self, *args, **kwargs):
+        """ iterator for testing that the provided checkpoint is model.ckpt-0 """
+        self.probe_checkpoint_path = kwargs["checkpoint_path"]
```

Comment:
I think this should be in `initialize` instead of `__init__`. See https://github.com/pesser/edflow/pull/62#discussion_r292514210

Reply:
should be solved in b22602b21a5623aaca08db6bb0829a206af88304